content/actions/learn-github-actions/finding-and-customizing-actions.md

```diff
@@ -64,6 +64,12 @@ The creators of a community action have the option to use tags, branches, or SHA
 
 You will designate the version of the action in your workflow file. Check the action's documentation for information on their approach to release management, and to see which tag, branch, or SHA value to use.
 
+{% note %}
+
+**Note:** Be especially careful with community actions that require secrets as inputs! It's probably better to [use SHAs](#using-shas) in that case. Even if you trust the maintainer, a successor or new team member could be less trustworthy.
```

Comment:
Thanks for this suggestion, @mktcode. We have a [Security hardening  guide](https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions) that explains things pretty well, so let's link over to that. 

```suggestion
**Note:** We recommend that you use a SHA value when using third-party actions. For more information, see [Security hardening for GitHub Actions](/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions)
```